### DISCUSSION TITLE
Increased computation time after Levante update

### DISCUSSION THREAD

**Discussion by ackerlar** [esm-tools/esm_tools#878] [HPCs] [answered]
Also, the computation time roughly doubled for my simulations after the latest Levante update. Please see the figure. I don't know where this sharp spike comes from but the computation time is doubled for all years I did since yesterday.

![compute_time](https://user-images.githubusercontent.com/72916819/206659901-f98a707c-a96f-4b22-b0ff-c5f435cd363f.png)

**Comment by mandresm** [comment]
Hi @ackerlar, I've only seen this now. Is it still the case?

**Reply by ackerlar** [reply, marked as answer]
Hi  @mandresm, no it has been fixed already

**Reply by mandresm** [reply]
thanks!
